HHS/OPRE-OPS
# feat: ✨ adds CAN detail page

## What changed

- adds Can detail page 
- updates CAN types
- adds routing and Tabs
- adds Tabs component

## Issue

#2542 

## How to test

1. goto Cans list
2. select 2023 from FY dropdown
3. click on a CAN from table
4. see detail page
5. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/44b38cfe-c050-421f-befc-67c0017de642

## Definition of Done Checklist
